packages/build/src/build-and-upload.spec.ts

```diff
@@ -50,20 +56,26 @@ describe('buildAndRelease', () => {
 
     tarballFile = { path: 'path', contentType: 'application/gzip' };
     compileAndZipExecutable = sinon.stub().resolves(tarballFile);
+    barque = createStubBarque();
     uploadToEvergreen = sinon.spy();
     uploadToDownloadCenter = sinon.spy();
     githubRepo = createStubRepo();
   });
 
   [true, false].forEach((isPublicRelease) => {
-    it(`uploads the artifact to evergreen if is ${isPublicRelease ? 'a' : 'not a'} public release`, async() => {
+    it(`uploads the artifact to evergreen if is ${isPublicRelease ? 'a' : 'not a'} public release`, async () => {
```

Comment:
I think we need a test to check we only release to barque if `isPublicRelease` 
and one to check that we don't if is not a public release or?

Reply:
are you saying you want a test that's like [this](https://github.com/mongodb-js/mongosh/pull/286/files#diff-6004d7618dcdb4b65e3f614bc3b71e98R119) but for `does not release to barque if not a public release`? Just triple checking!

Reply:
oh yep, thanks!